data-sources/mygene-info.yaml

```diff
@@ -1,7 +1,7 @@
 id: mygene-info
 source: MyGene.info
 source-link: 'http://mygene.info'
-status: incomplete
+status: complete
 description: 'MyGene.info provides a simple-to-use REST web services to query/retrieve gene annotation data.'
```

Comment:
@lwinfree Is the description a direct copy? The text I found drops the indefinite article (or maybe they had a typo earlier): `MyGene.info provides simple-to-use REST web services to query/retrieve gene annotation data. `

Reply:
can't remember...might be a typo on my end